jsk_panda_robot/jsk_panda_teleop/config/rviz/dual_panda.rviz

```diff
@@ -0,0 +1,680 @@
+Panels:
```

Comment:
Do you mean https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a41443/jsk_panda_robot/jsk_panda_startup/config/rviz/dual_panda_play.rviz?
They should be different because `jsk_panda_teleop/config/rviz/dual_panda.rviz` includes teleop-specific information (ON/OFF of haptic devices, commanded poses from the devices, ...):
![](https://user-images.githubusercontent.com/43567489/159150327-5e4d246b-2311-4eb4-814a-7f6fd11b6f29.png)
However, I totally forgot to test `dual_panda_play.rviz`, so I'll check this.

Reply:
Finally, I decided to keep both `jsk_panda_startup/config/rviz/dual_panda_play.rviz` and `jsk_panda_teleop/config/rviz/dual_panda.rviz`.
`jsk_panda_teleop/config/rviz/dual_panda.rviz`: Includes teleop-specific topics and translucent robot model seen from behind (operator-friendly view)
`jsk_panda_startup/config/rviz/dual_panda_play.rviz`: Excludes teleop-specific topics, robot model is big, opaque, and seen from diagonal (developer-friendly view)